lib/index.js

```diff
@@ -32,23 +32,25 @@ var Heap = module.exports = integration('Heap')
 
 Heap.prototype.initialize = function() {
   window.heap = window.heap || [];
-  window.heap.load = function(appid, config) {
-    window.heap.appid = appid;
-    window.heap.config = config;
-
-    var methodFactory = function(type) {
-      return function() {
-        window.heap.push([type].concat(Array.prototype.slice.call(arguments, 0)));
+  if (!window.heap.push) {
```

Comment:
Reflecting on this – it's better if it's `window.heap.appId`

The stubbed Heap could still be in the process of loading when this hits, so it's possible for `push` to be available for a small amount of time.